filters/special_casing/filter.py

```diff
@@ -0,0 +1,20 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class TextSpecialCasingFilter(SentenceOperation):
+    """Filter if any of the special casings appear in the sentence:
+    All text is capitalized or All text is lowercased or there is Title Casing in the Sentence.
+    """
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
```

Comment:
This should work for most languages, not just English, right?

Reply:
Yes, any Latin or Cyrillic script based language. Where could I get the language codes that we are using in the repo?

Reply:
I don't believe it's specified anywhere, but that's the convention I've seen used in all the transformations I've reviewed so far.

Reply:
Thanks, I have added multiple languages (using Latin, Cyrillic scripts) as the supported languages as well.